wies/projects/templatetags/projects_tags.py

```diff
@@ -0,0 +1,35 @@
+from django import template
+from ..models import Assignment, Ministry
+
+register = template.Library()
+
+@register.simple_tag
+def get_clients():
```

Comment:
this is only used by `simple_filter_modal.html`, which is not used by the application. what is its purpose?